### PR TITLE
Add support for Altium implied leading zeros omitted

### PR DESCRIPTION
**Pull request by sjgallagher2**
Altium generates files with format specification `%FSAX...Y...%` (no `L` for leading zero omission). This change updates the grammar to accept an empty Zeros parameter defaulting to lead zero omission.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -302,3 +302,6 @@ output.png
 .reference-assets.lock
 output.svg
 formatted.gbr
+
+# Local test files
+test/local/
```
Comment (Argmaster) [inline .gitignore:307]: Why are you including this entry?

**Review by Argmaster — COMMENTED** [review]
Hi @sjgallagher2 

Could you please include a test case which checks if this feature works correctly for purpose of regression testing?

**Comment by Argmaster** [comment]
Please also address issues reported by CI

File: .gitignore (hunk 1 above)
Comment (sjgallagher2) [reply .gitignore:307]: This was a folder I was using for my own test scripts, but the .gitignore can be reverted if it's just clutter

**Comment by sjgallagher2** [comment]
Previously worked around using Poetry but now have everything sorted, fixed up the tree and removed the local/ directory from .gitignore, hook checks all pass, just need to do the test case. 

I'm new to github and pull requests, is it easier to just close this PR and open a new one with a correct branch (implied-lead-2)? I worked there from origin/main to avoid having to undo things and get into git-fu. Or better to push changes right to this branch? 

For the test, I made some random geometry and generated this gerber (zipped for github):
[altium1.zip](https://github.com/user-attachments/files/17816471/altium1.zip)
How should I add a test for this? Add the altium test file to `examples/` and the `ExamplesEnum` and add a new test to `test/examples/gerberx3/api`? 

The test:

```python
from pygerber.gerber.api import GerberFile, Style

from pygerber.examples import ExamplesEnum, get_example_path

path_to_gerber_file = #get_example_path(ExamplesEnum.Altium_01)   #<- depends on test file zipped above

image = GerberFile.from_file(path_to_gerber_file).render_with_shapely(
    Style.presets.BLACK_WHITE
)
image.save("output.svg")

```

**Comment by sjgallagher2** [comment]
Thanks so much for the detailed help, I think I've got everything in order now, tests pass. 

PS. I noticed that [this page](https://argmaster.github.io/pygerber/3.0.0a3/90_development/00_setup.html) says `poe run-unit-tests` but I think it should be `poe test-unit` since the first didn't work for me. Worth opening a ticket?

**Comment by Argmaster** [comment]
> Thanks so much for the detailed help, I think I've got everything in order now, tests pass.

Sure, no problem, we have to wait for the results on latest commit though.

> PS. I noticed that [this page](https://argmaster.github.io/pygerber/3.0.0a3/90_development/00_setup.html) says `poe run-unit-tests` but I think it should be `poe test-unit` since the first didn't work for me. Worth opening a ticket?

Well, It would be worth opening a ticked, but I already updated contribution guidelines yesterdat, you can find [it here, on `dev` version](https://argmaster.github.io/pygerber/dev/90_development/40_contribution_guidelines.html).

**Review by Argmaster — APPROVED** [review]

**Pull request merged**